README.md

```diff
@@ -26,6 +26,7 @@ open source.
 * [Flutter documentation](https://docs.flutter.dev/)
 * [Development wiki](https://github.com/flutter/flutter/wiki)
 * [Contributing to Flutter](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md)
+* [Shorebird's customizations](./shorebird.md)
```

Comment:
In our other repos we have a shorebird/ dir and then docs/ inside that.  Or just README.md inside that.  This is also find though. 🤷